README.md

```diff
@@ -155,6 +155,12 @@ It is possible to both implement copy-on-write `ArrayBuffer`s and keep the "fixe
 
 At this time, Google Chrome deems this mitigation important enough for security to not implement CoW `ArrayBuffer`s.
 
+### How is `transfer` is used in browsers and in runtimes?
```

Comment:
I think this section warrants some qualification, since the first two links are just about detaching, which is half of the semantics of `transfer`.